src/pages/_document.tsx

```
@@ -1,10 +1,29 @@
 const Document = () => {
   return (
     <Html lang="en">
       <Head>
+      <Script
+          src="https://www.googletagmanager.com/gtag/js?id=G-4TKBWDD7DG"
+          strategy="afterInteractive"
+          async
+        />
+        <Script
+          id="gtag-init"
+          dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)